### ISSUE TITLE
Blocks are not rendering correctly after enabling inlay hints

### ISSUE THREAD

**Issue by HannesNowak** [leodevbro/vscode-blockman#84]
When configuring the setting `"editor.inlayHints.enabled": "on"` the blocks aren't rendered correctly.
Here is an example with Java code:
![image](https://user-images.githubusercontent.com/62110797/175015569-373a77a0-0e7c-4e6c-a1a4-9d33fca5fd6d.png)
So the block isn't the right size after the inlay hints are added to the lines.

**Comment by leodevbro** [comment]
Yeah, that's the limitation of VS Code, not a bug of Blockman. VS Code API does not give extensions access to the pixel locations/dimensions of inlay hints, that's why I recommend to disable inlay hints in order to use Blockman.

**Comment by wuyudi** [comment]
I use 
![image](https://github.com/leodevbro/vscode-blockman/assets/30024051/5f0caba5-07d7-44cf-ab9f-19452a54e8a3)

now. it kind of works :smile: 

![image](https://github.com/leodevbro/vscode-blockman/assets/30024051/726717a2-f071-43f0-85e3-d94a43b44b56)
